{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/templates/{{ cookiecutter.project_name }}/home.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>sugardough - mozilla webdev</title>
+  </head>
+  <body>
+    <style>
+      @font-face {
+        font-family: 'Fira Sans';
+        font-style: normal;
+        font-weight: 400;
+        src: url('https://mozilla.org/media/fonts/FiraSans-Regular.eot?#iefix') ('embedded-opentype'), url('https://mozilla.org/media/fonts/FiraSans-Regular.woff') ('woff'), , url('https://mozilla.org/media/fonts/FiraSans-Regular.ttf') ('truetype');
+      }
+
+      body {
+        background: #1E1E21 radial-gradient(ellipse farthest-side at left top , #00549E 0px, transparent 100%) no-repeat scroll 0% 0% / auto 640px;
+        font-family: 'Fira Sans', 'Arial';
+        color: white;
+        margin: 0 auto;
+        width: 80%;
+        text-align: center;
+      }
+      header {
+        font-size: 1.2 rem;
+      }
+      footer {
+        margin-top: 2rem;
+        font-size: 0.9rem;
+      }
+    </style>
+    <header>
+      <h1>mozilla webdev</h1>
+    </header>
+    It works! Now build something.
+    <footer>
+      {{ "{{" }} thisyear() {{ "}}" }}
```

Comment:
I feel like a `raw` block would be cleaner but it's really just a matter of opinion.